File: c sharp/2152 - Sum of the Digits.cs
using System;

namespace Main
{
    class Program
    {
        static void Main(string[] args)
        {
            int cases = int.Parse(Console.ReadLine());
            string num, linea;
            int sum;
            for (int i = 0; i < cases; i++)
            {
                linea = Console.ReadLine();
                if (linea[0] == '-') num = linea.Substring(1);
                else num = linea;
                sum = 0;
                for (int j = 0; j < num.Length; j++)
                    sum += (num[j] - '0');
                Console.WriteLine(sum);
            }
        }
    }
}

File: Java/2152 - Sum of the Digits.java
import java.util.*;

public class COJ2 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int cases = sc.nextInt();
		String num, linea;
		int sum;

		for (int i = 0; i < cases; i++) {
			linea = sc.next();
			if (linea.charAt(0) == '-')
				num = linea.substring(1);
			else
				num = linea;
			sum = 0;
			for (int j = 0; j < num.length(); j++)
				sum += (num.charAt(j) - '0');
			System.out.println(sum);
		}
	}
}